### PR TITLE
BETA: Register krstevski.is-a.dev

### PR DESCRIPTION
**Pull request by l3069**
Added `krstevski.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Omxpro** [comment]
> i dont know how to proceed, please help <3

just wait for the maintainers to reply

File: CNAME
Comment (Stef-00012) [inline CNAME:1]: You can't add files to the root folder

File: domains/krstevski.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "l3069",
+        "email": "krstevski.leonid2018@gmail.com"
+    },
+    "record": {
+        "CNAME": "krstevski.github.io"
```
Comment (Stef-00012) [inline domains/krstevski.json:7]: I cannot reach your website (404)

File: domains/_github-pages-challenge-l3069.krstevski
Comment (Stef-00012) [inline domains/_github-pages-challenge-l3069.krstevski:1]: File must be a json file (`.json`)

File: CNAME
Comment (l3069) [reply CNAME:1]: where should i add them?

**Pull request closed without merging**